packages/core/src/common/props.ts

```diff
@@ -7,7 +7,7 @@
 
 import * as React from "react";
 
-import { Intent } from "./intent";
+import { IconName, Intent } from "./index";
```

Comment:
oops this is what VS Code suggests for autofix. i manually edited most of them but apparently not all!